package.json

```diff
@@ -43,14 +43,14 @@
   },
   "homepage": "https://github.com/brandonroberts/ngrx-store-freeze#readme",
   "peerDependencies": {
-    "@ngrx/store": ">=4.0.0 || ^6.0.0-beta.1"
+    "@ngrx/store": ">=4.0.0 || ^6.0.0"
```

Comment:
Just having `>=4.0.0` is sufficient to cover 6.x

Reply:
Ah... what was I thinking needs more ☕️